packages/gatsby-source-filesystem/package.json

```diff
@@ -27,7 +27,8 @@
     "@babel/cli": "^7.8.4",
     "@babel/core": "^7.8.7",
     "babel-preset-gatsby-package": "^0.3.1",
-    "cross-env": "^5.2.1"
+    "cross-env": "^5.2.1",
+    "mitt": "^1.2.0"
```

Comment:
Is this because gatsby-source-filesystem uses mitt and you haven't migrated that?

Reply:
There's a test that uses it to create an eventEmitter in its mocks. The api is the same and we don't want to publish a new package for this case so I decided to leave it. But the package.json wasn't referring to it and it should so I fixed it. So it's kind of unrelated to my changes.